### PR TITLE
docs(tutorials): add pull-over and pull-out scenario in tutorials page

### PR DESCRIPTION
**Pull request by Ericpotato**
## Description

add pull-over and pull-out scenario in tutorials page

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [X] I've confirmed the [contribution guidelines].
- [X] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [X] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [X] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: docs/tutorials/ad-hoc-simulation/planning-simulation.md [hunk 1]
```diff
@@ -103,6 +103,16 @@ After that, you can see `AUTONOMOUS` sign on `OperationMode` and `AUTO` button i
 
    ![parking-maneuver](images/planning/parking/parking-maneuver.png)
 
+### Pull-over and pull-out scenario
+
+1. Begin by positioning the ego-vehicle at the roadside.
+
+   ![pullover-pullout](images/planning/pullover-pullout/pullover-pullout.jpg)
+
+2. set a destination and then activate the vehicle.
```
Comment (shmpwk) [inline docs/tutorials/ad-hoc-simulation/planning-simulation.md:112]:
```suggestion
2. Set a destination and then engage the ego vehicle.
```

File: docs/tutorials/ad-hoc-simulation/planning-simulation.md [hunk 2]
```diff
@@ -103,6 +103,16 @@ After that, you can see `AUTONOMOUS` sign on `OperationMode` and `AUTO` button i
 
    ![parking-maneuver](images/planning/parking/parking-maneuver.png)
 
+### Pull-over and pull-out scenario
+
+1. Begin by positioning the ego-vehicle at the roadside.
```
Comment (shmpwk) [inline docs/tutorials/ad-hoc-simulation/planning-simulation.md:108]:
```suggestion
1. Begin by positioning the ego vehicle at the road shoulder.
```

File: docs/tutorials/ad-hoc-simulation/planning-simulation.md [hunk 3]
```diff
@@ -103,6 +103,16 @@ After that, you can see `AUTONOMOUS` sign on `OperationMode` and `AUTO` button i
...
+
+   ![pullover-pullout](images/planning/pullover-pullout/pullover-pullout.jpg)
+
+2. set a destination and then activate the vehicle.
+
+3. In a pull-over scenario, similarly position the ego-vehicle in a lane and set a destination to the roadside.
```
Comment (shmpwk) [inline docs/tutorials/ad-hoc-simulation/planning-simulation.md:114]:
```suggestion
3. In a pull-over scenario, similarly set the ego vehicle in a lane and set a destination on the road shoulder.
```

**Comment by shmpwk** [comment]
Please click Set DCO pass
image

**Comment by shmpwk** [comment]
FYI, we usually call road shoulder. https://autowarefoundation.github.io/autoware.universe/main/planning/behavior_path_goal_planner_module/

**Comment by Ericpotato** [comment]
> FYI, we usually call road shoulder. https://autowarefoundation.github.io/autoware.universe/main/planning/behavior_path_goal_planner_module/

Thank you ! I've resolved DCO errors, it should be working now. 
ಥ‿ಥ
It's not easy, but I think I am getting used to Github slowly

File: docs/tutorials/ad-hoc-simulation/planning-simulation.md [hunk 4]
```diff
@@ -103,6 +103,16 @@ After that, you can see `AUTONOMOUS` sign on `OperationMode` and `AUTO` button i
 
    ![parking-maneuver](images/planning/parking/parking-maneuver.png)
 
+### Pull-over and pull-out scenario
```
Comment (shmpwk) [inline docs/tutorials/ad-hoc-simulation/planning-simulation.md:106]:
```suggestion
### Pull over and pull out scenario
```

File: docs/tutorials/ad-hoc-simulation/planning-simulation.md [hunk 5]
```diff
@@ -103,6 +103,16 @@ After that, you can see `AUTONOMOUS` sign on `OperationMode` and `AUTO` button i
...
+
+1. Begin by positioning the ego vehicle at the road shoulder.
```
Comment (shmpwk) [inline docs/tutorials/ad-hoc-simulation/planning-simulation.md:108]:
```suggestion
1. In a pull out scenario, set the ego vehicle at the road shoulder.
```

File: docs/tutorials/ad-hoc-simulation/planning-simulation.md [hunk 6]
```diff
@@ -103,6 +103,16 @@ After that, you can see `AUTONOMOUS` sign on `OperationMode` and `AUTO` button i
...
+
+   ![pullover-pullout](images/planning/pullover-pullout/pullover-pullout.jpg)
+
+2. Set a destination and then engage the ego vehicle.
```
Comment (shmpwk) [inline docs/tutorials/ad-hoc-simulation/planning-simulation.md:112]:
```suggestion
2. Set a goal and then engage the ego vehicle.
```

File: docs/tutorials/ad-hoc-simulation/planning-simulation.md [hunk 7]
```diff
@@ -103,6 +103,16 @@ After that, you can see `AUTONOMOUS` sign on `OperationMode` and `AUTO` button i
...
+
+3. In a pull-over scenario, similarly set the ego vehicle in a lane and set a destination on the road shoulder.
```
Comment (shmpwk) [inline docs/tutorials/ad-hoc-simulation/planning-simulation.md:114]:
```suggestion
3. In a pull over scenario, similarly set the ego vehicle in a lane and set a goal on the road shoulder.
```

File: docs/tutorials/ad-hoc-simulation/planning-simulation.md [hunk 8]
```diff
@@ -103,6 +103,16 @@ After that, you can see `AUTONOMOUS` sign on `OperationMode` and `AUTO` button i
 
    ![parking-maneuver](images/planning/parking/parking-maneuver.png)
 
+### Pull over and pull out scenario
```
Comment (shmpwk) [inline docs/tutorials/ad-hoc-simulation/planning-simulation.md:106]:
```suggestion
### Pull out and pull over scenario
```

**Comment by shmpwk** [comment]
@Ericpotato 
A minor request. Could you please make the photo a horizontal rectangle? It would be more beautiful.

**Comment by Ericpotato** [comment]
> @Ericpotato A minor request. Could you please make the picture more horizontal rectangle? It would be more beautiful.

Done! Please check

**Comment by Ericpotato** [comment]
> A minor request. Could you please make the picture more horizontal rectangle? It would be more beautiful.

Done! Please check

**Comment by shmpwk** [comment]
@Ericpotato Could you check the file name correspondence?

**Comment by shmpwk** [comment]
@Ericpotato 
Sorry I failed to explain clearly but what "more horizontal rectangle" means is to make width all the same. 
image

**Review by shmpwk — APPROVED** [review]
LGTM

**Pull request merged**